_posts/2020-07-21-tips-and-tricks-for-running-strimzi-with-kubectl.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title:  "Tips & Tricks for running Strimzi with kubectl"
+date: 2020-07-21
+author: jakub_scholz
+---
+
+When running Apache Kafka on Kubernetes with Strimzi, one of the tools you might use very often is `kubectl`.
+You can use it for controlling your Kubernetes cluster and its resources.
+And because Strimzi is using Custom Resources to extend the Kubernetes APIs, it can use it also with Strimzi.
+In this blog post, we will have a look at some tips and tricks how to make it easier.
+
+<!--more-->
+
+## Working with Kubernetes resources
+
+`kubectl` can be used for all the basic tasks with the Strimzi custom resources.
+You can use all `kubectl` commands such as `get`, `describe`, `edit`, or `delete` coupled with the resource type.
+So for example `kubectl get kafkatopics` will get you a list of all Kafka topics or `kubectl get kafkas` to get all Kafka clusters.
+
+Most people know that for all of the resource types, you can use both singular and plural.
+So `kubectl get kafkas` gets you the same results as `kubectl get kafka`.
+But not everyone knows that the custom resources have also _short name_.
+So you can for example also do `kubectl get k`.
+
+```
+$ kubectl get k
+NAME         DESIRED KAFKA REPLICAS   DESIRED ZK REPLICAS
+my-cluster   3                        3
+```
+
+The following table shows all our resources and their short names:
+
+| Resource              | Long name         | Short name    |
+| --------------------- |:----------------- |:------------- |
+| Kafka                 | kafka             | k             |
+| Kafka Topic           | kafkatopic        | kt            | 
+| Kafka User            | kafkauser         | ku            |
+| Kafka Connect         | kafkaconnect      | kc            |
+| Kafka Connect S2I     | kafkaconnects2i   | kcs2i         |
+| Kafka Connector       | kafkaconnector    | kctr          |
+| Kafka Mirror Maker    | kafkamirrormaker  | kmm           |
+| Kafka Mirror Maker 2  | kafkamirrormaker2 | kmm2          |
+| Kafka Bridge          | kafkabridge       | kb            |
+| Kafka Rebalance       | kafkarebalance    | kr            |
+
+
+This is really handy because some of the resources have really long names. 
+So learning the short names can save a lot of time while operating Strimzi or working on new features.
+
+## Resource categories
+
+The custom resources can be also grouped into categories.
+The name of the category can be used in the `kubectl` commands instead of the individual resource types.
+All our custom resources are in the category `strimzi`.
```

Comment:
```suggestion
All Strimzi custom resources are in the category `strimzi`.
```
I know this is an informal article, but the possessive our might be confusing here.